<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CircAdapt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/resources.qrc">
    <normaloff>:/CircAdapt/icons/circadapt.png</normaloff>:/CircAdapt/icons/circadapt.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>false</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="GraphGrid" name="graphGrid" native="true"/>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="opaqueResize">
        <bool>false</bool>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>false</bool>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="LoopGraph" name="pvGraph" native="true"/>
        <widget class="LoopGraph" name="ssGraph" native="true"/>
       </widget>
       <widget class="ValueView" name="valueView" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExecute">
    <property name="title">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSignals">
     <property name="title">
      <string>Signals</string>
     </property>
     <addaction name="actionStress_strain"/>
     <addaction name="separator"/>
     <addaction name="actionPressures"/>
     <addaction name="actionVolumes"/>
     <addaction name="actionFlows"/>
     <addaction name="actionStress"/>
     <addaction name="actionStrain"/>
     <addaction name="actionM_mode"/>
     <addaction name="actionSaturation"/>
    </widget>
    <addaction name="menuSignals"/>
    <addaction name="separator"/>
    <addaction name="actionShow_oxygen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExecute"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionSnapshot"/>
   <addaction name="actionReset"/>
   <addaction name="actionAutoscale"/>
   <addaction name="actionParameter_Settings"/>
  </widget>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/CircAdapt/icons/media-play-32.png</normaloff>
     <normalon>:/CircAdapt/icons/media-pause-32.png</normalon>:/CircAdapt/icons/media-play-32.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/CircAdapt/icons/camera.png</normaloff>:/CircAdapt/icons/camera.png</iconset>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/CircAdapt/icons/reset.png</normaloff>:/CircAdapt/icons/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionAutoscale">
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/CircAdapt/icons/autoscale.png</normaloff>:/CircAdapt/icons/autoscale.png</iconset>
   </property>
   <property name="text">
    <string>Autoscale</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionParameter_Settings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/resources.qrc">
     <normaloff>:/CircAdapt/icons/settings_icon.png</normaloff>:/CircAdapt/icons/settings_icon.png</iconset>
   </property>
   <property name="text">
    <string>Parameter Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionStress_strain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stress-strain</string>
   </property>
  </action>
  <action name="actionPressures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pressure</string>
   </property>
  </action>
  <action name="actionVolumes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </action>
  <action name="actionFlows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flow</string>
   </property>
  </action>
  <action name="actionStress">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stress</string>
   </property>
  </action>
  <action name="actionStrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Strain</string>
   </property>
  </action>
  <action name="actionM_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>M-mode</string>
   </property>
  </action>
  <action name="actionShow_oxygen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show oxygen</string>
   </property>
  </action>
  <action name="actionSaturation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Saturation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoopGraph</class>
   <extends>QWidget</extends>
   <header>CircAdaptUI/Graphs/loopgraph.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphGrid</class>
   <extends>QWidget</extends>
   <header>CircAdaptUI/Widgets/graphgrid.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ValueView</class>
   <extends>QWidget</extends>
   <header>CircAdaptUI/Widgets/valueview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
